src/mame/heathkit/h89.cpp

```diff
@@ -53,6 +62,10 @@ class h89_state : public driver_device
 	void h89_mem(address_map &map);
 };
 
+void h89_state::init()
+{
+	m_tlb->init();
+}
```

Comment:
You *really* shouldn’t be doing this.  The device lifecycle takes care of your needs.